capstone/src/main/java/com/google/sps/servlets/AnnouncementsServlet.java

```diff
@@ -11,13 +11,16 @@
 /** Servlet that returns some example club content */
 @WebServlet("/announcements")
 public class AnnouncementsServlet extends HttpServlet {
+  private static final String CLUB_NAME_PROP = "name";
```

Comment:
Noticed "CLUB_NAME_PROP" is a static final variable you define in two files.

If we will end up having more of these types of variables would be good to create a constants file to have them defined in one place. Then the file can just be imported and variables referenced from there.

+Chris if you have any more thoughts on this?

Reply:
+1 to making a separate file for constants that are shared between servlets

Reply:
Got it, thanks!